fluentd/datadog_checks/fluentd/fluentd.py

```diff
@@ -20,12 +20,14 @@ def __init__(self, name, init_config, instances):
         super(Fluentd, self).__init__(name, init_config, instances)
         if not ('read_timeout' in self.instance or 'connect_timeout' in self.instance):
             # `default_timeout` config option will be removed with Agent 5
-            self.http.options['timeout'] = (
+            timeout = (
                 self.instance.get('timeout')
+                or self.instance.get('default_timeout')
                 or self.init_config.get('timeout')
                 or self.init_config.get('default_timeout')
                 or self.DEFAULT_TIMEOUT
             )
+            self.http.options['timeout'] = (timeout, timeout)
```

Comment:
why a tuple if later is casted into a float? (`connect_timeout = read_timeout = float(config['timeout'])`)

Reply:
The `[timeout](
https://github.com/DataDog/integrations-core/blob/9ef31f7fb53fc70d66253d87a90c6b9b474e8853/datadog_checks_base/datadog_checks/base/utils/http.py#L249)` options in the wrapper are a tuple to support `read_timeout` and `connect_timeout` options.

```
            'timeout': (connect_timeout, read_timeout),
```